packages/browser-integration-tests/suites/replay/extendNetworkBreadcrumbs/fetch/captureRequestBody/init.js

```diff
@@ -4,9 +4,9 @@ window.Sentry = Sentry;
 window.Replay = new Sentry.Replay({
   flushMinDelay: 200,
   flushMaxDelay: 200,
-  _experiments: {
-    captureNetworkBodies: true,
-  },
+
+  networkDetailAllowUrls: ['http://localhost:7654/foo'],
```

Comment:
Just double checking, is the port here stable?

Reply:
🤷 this is also in the tests, so it would fail there as well (we check the url of the events is this in there)